Build Error due to Broken Links

While executing the `npm run build` command, the build process failed due to broken links detected by Docusaurus. The error message indicates that there are broken links on pages `/sdks/api` and the Home Page (`/`). The detailed error message is as follows:

[ERROR] Unable to build website for locale zh-Hans.
[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

On source page path = /sdks/api:
-> linking to docs/sdks/api/msg/index.mdx (resolved as: /sdks/docs/sdks/api/msg/index.mdx)

On source page path = /:
-> linking to /restapi/userManagement/userRegister

at throwError (C:\Users\smile\Desktop\web-docs\openim-docs\node_modules@docusaurus\logger\lib\index.js:76:11)
...


#### Steps to Reproduce:
1. Clone the repository.
2. Navigate to the project directory.
3. Run `npm install` to install the dependencies.
4. Run `npm run build` to build the documentation.

#### Expected Result:
The documentation should build without any errors.

#### Actual Result:
The build process fails due to broken links.

#### Suggested Fix:
- Verify and update the paths for the broken links on the mentioned pages.
- Additionally, temporarily setting `onBrokenLinks: 'ignore'` in `docusaurus.config.js` allows the build to pass, although it's more advisable to fix the broken links for a better user experience.

#### Environment:
- Docusaurus version: (Specify the version of Docusaurus you are using)
- Node.js version: 14
- OS: Windows (Specify your OS and its version)

#### Additional Context:
Add any other context about the problem here.

#### Attachments:
![image](https://github.com/openimsdk/docs/assets/86140903/fda0dd34-5bd5-4c9c-a6c1-2a0586fdf12f)

--- a/.github/workflows/main-ci.yaml
+++ b/.github/workflows/main-ci.yaml
@@ -11,12 +11,12 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
 
     - name: Setup Node.js
-      uses: actions/setup-node@v2
+      uses: actions/setup-node@v4
       with:
-        node-version: '14'
+        node-version: 'latest'
 
     - name: Install dependencies
       run: npm install

--- a/.github/workflows/main-ci.yaml
+++ b/.github/workflows/main-ci.yaml
@@ -24,11 +24,10 @@ jobs:
     - name: Build documentation
       run: npm run build
 
-    - name: Run tests
-      run: npm test  # Replace with your test script if different
-
     - name: Lint check
       run: npm run lint
+      continue-on-error: true
 
     - name: Type checking
       run: npm run typecheck
+      continue-on-error: true

